README.md

```diff
@@ -1,17 +1,36 @@
-## My Project
-
-TODO: Fill this README out!
-
-Be sure to:
-
-* Change the title in this README
-* Edit your repository description on GitHub
-
-## Security
-
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+## amazon-kinesis-video-streams-signaling
+
+The goal of the Signaling library is to enable control plane communication
```

Comment:
is this just for control plane?

Reply:
Corrected.

Reply:
the TURN servers (https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_Operations_Amazon_Kinesis_Video_Signaling_Channels.html) and the Websocket APIs belong to data plane